code/game/objects/structures/crates_lockers/closets/walllocker.dm

```diff
@@ -28,4 +28,16 @@
 		/obj/item/clothing/head/helmet/space = 2,
 		/obj/item/clothing/suit/space = 2,
 		/obj/item/tank/oxygen = 2
-	)
\ No newline at end of file
+	)
+
+/obj/structure/closet/walllocker/emerglocker
+	name = "emergency locker"
+	desc = "A wall mounted locker with emergency supplies."
+	closet_appearance = /decl/closet_appearance/wall/emergency
+
+/obj/structure/closet/walllocker/emerglocker/WillContain()
```

Comment:
If you look at how this type is implemented on Polaris, it's basically just `/obj/structure/emergency_dispenser`. Is this change (to be an actual closet) intentional?

Reply:
They're terribly written but almost exactly the same as the emerglocker type, just without the crowbar.